### PR TITLE
Added plot and 2 cards

### PR DESCRIPTION
**Pull request by otecMuxah**
[OLGA-40](https://youtrack.raccoongang.com/issue/OLGA-40)
[OLGA-28](https://youtrack.raccoongang.com/issue/OLGA-28)
[OLGA-25](https://youtrack.raccoongang.com/issue/OLGA-25)

### CODE REVIEW

File: web/olga/charts/static/charts/graphs.js [hunk 1]
```diff
@@ -119,8 +119,51 @@
...
+        let studNum = [];
+        let certNum = [];
+        let enthNum = [];
+        for (let item in newData.monthly) {
+            studNum.push(newData.monthly[item][0]);
+            certNum.push(newData.monthly[item][1]);
+            enthNum.push(newData.monthly[item][2]);
+        }
+
+
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: Too many empty lines, please leave not more than two.

File: web/olga/charts/static/charts/graphs.js [hunk 2]
```diff
@@ -119,8 +119,51 @@
...
+        var layout = {barmode: 'group'};
+
+        Plotly.newPlot(chart, data, layout);
+    }
+
     appendChartData(instances_gd, [instances,courses,students], 'Instances, Courses, Students');
-    
+    appendSecChart(courses_gd, [], 'Need_a_name');
+
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: ditto

**Review by idegtiarov — CHANGES_REQUESTED** [review]
Some style nits changes are desirable

File: web/olga/charts/static/charts/graphs.js (hunk 1 above)
Comment (otecMuxah) [reply web/olga/charts/static/charts/graphs.js]: done

File: web/olga/charts/static/charts/graphs.js (hunk 2 above)
Comment (otecMuxah) [reply web/olga/charts/static/charts/graphs.js]: done

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**